### ISSUE TITLE
5.3 Changing Governance intro vs. 5.4.3 Meeting Process

### ISSUE THREAD

**Issue by chrcowan** [holacracyone/Holacracy-Constitution#401]
I think 5.3 should probably call out explicitly that it's intended for asynchronous governance, or governance outside of meetings. But not sure if I'm reading it right, so I'm posting this as an issue rather than a suggestion. 

The issue I ran into was that I was trying to prepare resources for Facilitators on 5.0, and so I was comparing and contrasting 5 to 4.1, when I mistakenly thought that 5.3, and its generic language about changing governance was somehow supposed to be equivalent and replacement for, the specific meeting steps used in 4.1 (specifically section 3.3.5).  

Upon further investigation, I realized (duh) version 5 still has a specific governance meeting section (5.4), and that version 4.1 also has a more generic/intro section for "changing governance" (3.2). 

So, I went back to figure out how I could have made such a simple mistake, and I noticed that the language of v.5 section 5.3 says, "Any Circle Member of a Circle **may initiate its Governance Process** by proposing a change to its Governance," whereas the 4.1 version says, "Any Core Circle Member of a Circle may propose changing its Governance, thus making a “Proposal” as a “Proposer”.

I'm assuming the change in v5 was to give clearer constitutional authority to asynchronous governance, and in fact, v5. section 5.4 (which is about the in-meeting/live governance meetings ) seems to contrast the generic/intro "changing governance" section by saying, "**In addition to processing Proposals made asynchronously outside of meetings**, each Circle will also hold regular..." but the thing is, nothing is ever explicitly said about asynchronous governance in the generic/intro of v.5 section 5.3!

So, basically, I think I made the mistake because the language tweaks in v5 language including "may initiate its Governance Process," is too ambiguous without more clearly calling out "asynchronous governance." I think others could easily make the same mistake.

**Comment by brianjrobertson** [comment]
@chrcowan That section is not just for asynchronous governance - the rules in there apply to any governance change regardless of the forum it's made in.  In v4.1, the in-meeting process was the main focus, with a tacked-on rule for doing things outside of meetings; but in v5.0, the forum is de-emphasized - now the rules are just about how to change governance more broadly, regardless of the forum; you could even use them to change governance in a meeting of all relevant parties that wasn't officially a "Governance Meeting"...

**Comment by SamirSaidani** [comment]
If I'm understanding it right, it means that in a Tactical Meeting with all relevant parties present in the meeting, anyone could trigger a Governance Process. Do you confirm ?

**Comment by brianjrobertson** [comment]
@samir-s Yes; although I typically wouldn't advise trying that, especially given that a tactical agenda item is timeboxed and can be cut off by the facilitator at the end of the allotted time.  But it is technically allowed by the constitution now, as long as all Circle Members are present in the tactical.

**Comment by brianjrobertson** [comment]
@chrcowan I'm closing this, although please do comment to reopen if you have anything more you want to dig into or suggest in response on my comments above.
